components/FormInput/FormInput.css

```diff
@@ -43,6 +43,8 @@
   transition: all 200ms ease;
   background: var(--color-error);
   color: white;
+  /* Stops the element being bigger than the input */
+  box-sizing: border-box;
```

Comment:
This should be set globally... If it's not we should chuck it in `Styleguide.css`